### ISSUE TITLE
Square brackets of `tag` arguments in the log methods

### ISSUE THREAD

**Issue by AzariasB** [libktx/ktx#370]
Hi,
first thanks a lot for this awesome lib, which makes it quite enjoyable to make a libgdx app with kotlin.

Currently when using the logging helpers : 
```kotlin
info { "Log with default tag" }
```
On desktop, the following is printed : 
```
[[INFO] ] Log with default tag
```

Libgdx takes care of adding brackets when printing to the console for the desktop version [see here](https://github.com/libgdx/libgdx/blob/master/backends/gdx-backend-lwjgl3/src/com/badlogic/gdx/backends/lwjgl3/Lwjgl3ApplicationLogger.java#L28)

I haven't tested for android or other platforms, but my guess is that the square bracket is also added by libgdx.

What I would expect when using the logging methods provided by ktx is that the default tags don't contain any brackets.

Hopefully that's clear :)

**Comment by czyzby** [comment]
It's possible that the behavior changed over the time, since I remember testing this module on desktop and it seemed fine. Thanks for pointing this out.

**Comment by czyzby** [comment]
I've changed all of the default tags not to include any brackets or whitespaces. This should work fine on most platforms.

Also, I've noticed that the `Logger` class implementation can be improved in terms of customization of the message format. It now includes a `buildMessage` method that can be overridden to modify how the messages are logged, e.g. by appending current time.

The changes should be available shortly via the snapshot release (`1.10.0-SNAPHOT`). I'll go through the existing issues this week, and might do a stable release soon.
